content/1_General/Running_Code_Online.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 id: running-code-online
 title: Running Code Online
-author: Benjamin Qi, Hankai Zhang, Anthony Wang, Nathan Wang, Nathan Chen
```

Comment:
what did they do to deserve this

Reply:
i think stuff they wrote were moved to a different module